ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -549,3 +554,37 @@ def test_paddle_cos(
         frontend=frontend,
         on_device=on_device,
     )
+  
+# sqrt
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="sqrt",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
You should be drawing `helpers.get_dtypes("valid")` here. The test handler will find the supported dtypes taking into account the `with_supported_dtypes` decorator you added to the function.

Reply:
I am on it